shared/bitutil/bit.go

```diff
@@ -83,3 +78,15 @@ func fillBit(target byte, index uint64) byte {
 	bitShift := 7 - index
 	return target ^ (1 << bitShift)
 }
+
+// BitfieldBit Extract the bit in `bitfield` at position `i`.
```

Comment:
Please fix function comment

```suggestion
// BitfieldBit extracts the bit in bitfield at input position i.
```

Reply:
thanks